CHANGES

```diff
@@ -5,6 +5,7 @@
 - API Change: Net#openURI now returns a boolean to indicate whether the uri was actually opened.
 - API Change: DefaultShader now always combines material and environment attributes
 - API Change: Added ShapeRenderer constructor to pass a custom shader program to ImmediateModeRenderer20.
+- API Change: Added ClasspathFileHandleResolver for resolving assets via Gdx.files.classpath(string).
```

Comment:
You might be right, there's already a line close to the start of the1.5.5 version notes that starts with "Added". Might as well drop the "API Change: ". What do you think?